src/js/components/SnippetForm/SnippetEditor/SnippetEditor.tsx

```diff
@@ -109,14 +109,12 @@ const SnippetTypeTabs: React.FC<SnippetTypeTabsProps> = ({
 				? <a
 					className="button button-large nav-tab-button nav-tab-inactive go-pro-button"
 					href="https://codesnippets.pro/pricing/"
-					title="Find more about Pro"
 					onClick={event => {
 						event.preventDefault()
 						openUpgradeDialog()
 					}}
 				>
-					{_x('Upgrade to ', 'Upgrade to Pro', 'code-snippets')}
-					<span className="badge">{_x('Pro', 'Upgrade to Pro', 'code-snippets')}</span>
+					{__('Upgrade to <span class="badge">Pro</span>', 'code-snippets')}
```

Comment:
This will merge similar translation strings, and reduce the total strings by 2:

![image](https://github.com/user-attachments/assets/88cd5fda-39bd-4057-a05b-5a55a49c17f6)